edgelet/edgelet-core/src/module.rs

```diff
@@ -56,7 +55,7 @@ pub struct ModuleRuntimeState {
     started_at: Option<DateTime<Utc>>,
     finished_at: Option<DateTime<Utc>>,
     image_id: Option<String>,
-    pid: Pid,
+    pid: Option<i32>,
```

Comment:
Why remove Pid? Could you have done Option<Pid> here?

Reply:
I tried to get rid of Pid wherever I could because it's very specific to docker authentication. Pid in current implementation has `Pid::Any` value, and obviously, a container cannot have this value. So Options<i32> works better here